services/frontend/pkg/config/config.go

```diff
@@ -138,7 +142,7 @@ type ResourceInfoCaches struct {
 
 // RedisDriver holds redis configuration
 type RedisDriver struct {
-	Address  string `yaml:"address" env:"FRONTEND_OCS_RESOURCE_INFO_CACHE_REDIS_ADDR" desc:"A comma separated list of addresses to access the configured store. This has no effect when the 'memory' store is configured. Note that the behaviour how addresses are used is dependent on the library of the configured store."`
+	Address  string `yaml:"address" env:"FRONTEND_OCS_RESOURCE_INFO_CACHE_REDIS_ADDR" desc:"A comma separated list of nodes to access the configured store. This has no effect when 'memory' or 'ocmem' stores are configured. Note that the behaviour how nodes are used is dependent on the library of the configured store."`
```

Comment:
`FRONTEND_OCS_RESOURCE_INFO_CACHE_REDIS_ADDR`
`_ADDR` shoud'nt that be `_NODES`

Reply:
The `FRONTEND_OCS_RESOURCE_INFO_CACHE` vars have been adapted by now as well.

Regarding the auth information for redis those are to be given as part of the node address consistently now, e.g. `user:password@localhost:6789`.